packages/atomic/cypress/integration/facets/facet-utils.ts

```diff
@@ -279,3 +279,22 @@ export function assertShowLessUA(field: string) {
     expect(analyticsBody.customData).to.have.property('facetField', field);
   });
 }
+
+export function typeQueryAndWaitUA(searchboxSelector: string, query: string) {
```

Comment:
could we have a new facet-search-utils file for these?

Reply:
facetSearch is/ and will be included in same test suites of Facets, and so far I don't see if we will have much function for that particular facet-search, so I suggest keep like this for now :)